test/endpoints/mockAndelapi.test.js

```diff
@@ -0,0 +1,26 @@
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import chai, { expect } from 'chai';
+import app from '../../server';
+import slackMocks from '../mocks/slack';
+import { slackClient } from '../../server/modules/slack/slackIntegration';
+
+chai.use(chaiHttp);
+const fakeUserList = sinon.stub(slackClient.users, 'list').callsFake(() => slackMocks.userList);
+
+describe('Tests for mockAndelaApi endpoint\n', () => {
+  it('Should return list of mock placements with a 200 status code', (done) => {
+    const status = 'External Engagements - Rolling Off';
+    chai
+      .request(app)
+      .get(`/mock-api/api/v1/placements/?status=${status}`)
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        expect(res.body)
+          .to.have.property('values')
+          .to.be.an('array');
```

Comment:
I think that we should also check that the array count of the values property in response is equal to the count of number of users with emails in slackMocks list since the mock function uses the emails of users from the slacklist to generate fake placement data.

Reply:
It cant be tested because the mock users is randomly generated from the users in work space. Can be between 0-4 placements per automation

Reply:
what if you mock the random number used in your for loop
```javascript
for (let index = 0; index < faker.random.number(5); index++) {
```
if you can mock faker.random.number(5); it once to a specific value so that it would be predictable
The reason for such a test is to guarantee that it not just returning a 200 response but it returning the right values data

Reply:
or check that in the response value a user whose email is in the slack list exists

Reply:
I have written test to confirm that the array in the response has length greater than `0` and less than `6`

Reply:
nice implementation